javaagent/src/test/groovy/io/opentelemetry/javaagent/classloading/ShadowPackageRenamingTest.groovy

```diff
@@ -5,7 +5,7 @@
 
```

Comment:
@trask @mateuszrzeszutek Can you confirm my changes to this file? It was needed since no more Guava in agent.